src/Bicep.Core/FileSystem/PathHelper.cs

```diff
@@ -137,6 +140,30 @@ public static Uri FilePathToFileUrl(string filePath)
             return uriBuilder.Uri;
         }
 
+        public static bool TryGetUsingPath(UsingDeclarationSyntax? usingDeclarationSyntax, [NotNullWhen(true)]out string? bicepPath, [NotNullWhen(false)]out DiagnosticBuilder.ErrorBuilderDelegate? failureBuilder)
+        {
+            var pathSyntax = usingDeclarationSyntax?.TryGetPath();
+            if (pathSyntax == null)
+            {
+                bicepPath = null;
+                failureBuilder = x => x.TemplatePathHasNotBeenSpecified();
```

Comment:
I think we can remove `TemplatePathHasNotBeenSpecified` and set `failureBuilder` to `null`. The parser already generates a similar error if the path syntax is missing.

Reply:
Can you link to where that is being generated? Are you referring to ```ParamsParser```?

Reply:
Yes. It's generated here: https://github.com/Azure/bicep/blob/Bicep-params-file/src/Bicep.Core/Parsing/ParamsParser.cs#L88, however I realized that we should not reuse `ExpectedModulePathString` in the parser. @cqthyy Could you create a new error diagnostic for the case where the Bicep file path is missing in a using declaration and update the `ParamsParser` to use it?